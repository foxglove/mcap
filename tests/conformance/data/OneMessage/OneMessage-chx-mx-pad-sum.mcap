version https://git-lfs.github.com/spec/v1
oid sha256:70ba72baaa09ed2ae6608e7c337a58014127de9fadddbef99b4b363d8fa9df10
size 271
