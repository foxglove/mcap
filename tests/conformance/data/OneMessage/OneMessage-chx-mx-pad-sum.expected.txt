Header profile="" library=""
ChannelInfo id=1 topic=example message_encoding=a schema_encoding=c schema=b schema_name=Example metadata={foo=bar}
Message channel_id=1 sequence=10 publish_time=1 log_time=2 message_data=<010203>
SummaryOffset group_opcode=12 group_start=147 group_length=0
SummaryOffset group_opcode=9 group_start=147 group_length=0
SummaryOffset group_opcode=7 group_start=147 group_length=0
Footer summary_start=0 summary_offset_start=147
